src/main/java/me/zeroeightsix/kami/module/modules/movement/ElytraReplace.kt

```diff
@@ -25,137 +29,238 @@ class ElytraReplace : Module() {
     private val elytraFlightCheck = register(Settings.b("ElytraFlightCheck", true))
     private val logToChat = register(Settings.booleanBuilder("MissingWarning").withValue(false).build())
     private val playSound = register(Settings.booleanBuilder("PlaySound").withValue(false).withVisibility { logToChat.value }.build())
-    private val logThreshold = register(Settings.integerBuilder("MissingThreshold").withRange(1, 10).withValue(2).withVisibility { logToChat.value }.build())
+    private val logThreshold = register(Settings.integerBuilder("LogThreshold").withRange(1, 10).withValue(2).withVisibility { logToChat.value }.build())
```

Comment:
why was this renamed?

Reply:
I felt that it would be a bit clearer to have the name in the GUI match the variable in the code. IMO `LogThreshold` conveys the idea of "the limit before elytra swaps start being logged in chat" better than `MissingThreshold`as elytras don't really go "missing".  `SwapThreshold` could work as well as reverting the change if its important to change names as little as possible.

Reply:
It's supposed to be the amount of elytras you have left before it starts warning you in the chat. I'd just prefer a more intuitive name which isn't too long

Reply:
Do any of the following names work?
`ElytrasLeft`, `ElytraThreshold`, `ElytraLogLimit`.
If none of them work I can just revert it, no problems here!

Reply:
I think `WarningThreshold` is most clear?

Reply:
Ah, that seems much clearer! I'll edit it asap 👍